server/openapi_server/models/meeting.py

```
@@ -0,0 +1,51 @@
+                    'filename': "agenda.pdf",
+                    'url': "http://example.com/agenda.pdf"
+                }]
+            }
+        }
+
+    @classmethod
+    def from_dict(cls, dikt) -> 'Meeting':
+        """Returns the dict as a model, ensuring types are correct."""
+        return cls(**dikt)
+
+    class Config:
```

Ruff (0.11.9)

Redefinition of unused `Config` from line 27

(F811)